oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/AlarmModuleProvider.java

```diff
@@ -91,4 +92,8 @@ public void notifyAfterCompleted() throws ServiceNotProvidedException, ModuleSta
             ConfigurationModule.NAME
         };
     }
+
+    ModuleManager getModuleManager() {
+        return getManager();
+    }
 }
```

Comment:
This method is only for testing purpose, we don't add methods just for testing, please use `Whitebox.getInternalState(moduleProvider, "manager");` to get the `manager` field in the unit test

Reply:
> This method is only for testing purpose, we don't add methods just for testing, please use `Whitebox.getInternalState(moduleProvider, "manager");` to get the `manager` field in the unit test

copy that. 

> Also update the `CHANGES.log` to include this feature

copy that .